README.md

```diff
@@ -1 +1,27 @@
-# NumberPlateRecognition
\ No newline at end of file
+# NumberPlateRecognition
+
+This projects aims to create a program to make images and videos safe to upload under strict data protection laws.
+
+# Developer Information
+
+## Rules
+
+- Always create a new brach for a new feature. Use the first letter of your name as the start of the branch name (e.g. c-topic)
+- Use typing for all functions and classes
+
+## Conda Environment
+
+All the dependencies for this project are saved in the environments.yml file for a conda environment. Use 
+```
+conda env create --name NumberPlateRecognition --file=environments.yml
```

Comment:
there is a s to much in enviroments in the command conda env create --name NumberPlateRecognition --file=environments.yml.
It has to be conda env create --name NumberPlateRecognition --file=environment.yml

Reply:
Thanks! I changed it.